java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1961,6 +1964,75 @@ public T lastsave() {
         return getThis();
     }
 
+    /**
+     * Displays a piece of generative computer art and the Redis version.<br>
+     * The command will be routed to a random node.
```

Comment:
Some of these docs mention the command will be routed to a random node and others do not, should they all mention this?

Reply:
Oh, it shouldn't be mentioned in transcation at all, thanks